examples/MotorImagery/two_class_motor_imagery.py

```diff
@@ -12,29 +12,41 @@
 from moabb.analysis import analyze
 
 import mne
+import numpy as np
 mne.set_log_level(False)
 
 import logging
 import coloredlogs
-logging.basicConfig(level=logging.INFO)
+logging.basicConfig(level=logging.DEBUG)
 logger = logging.getLogger()
-coloredlogs.install(level=logging.INFO)
+coloredlogs.install(level=logging.DEBUG)
 
 datasets = utils.dataset_search('imagery', events=['right_hand', 'left_hand'],
                                 has_all_events=True, min_subjects=2,
                                 multi_session=False)
 
-paradigm = LeftRightImagery()
+for d in datasets:
+    d.subject_list = d.subject_list[:10]
+
+paradigm = LeftRightImageryMultiPass(fbands=np.array([[8, 12],
+                                                      [12, 16],
+                                                      [16, 20],
+                                                      [20, 24],
+                                                      [24, 28],
+                                                      [28, 32]]))
 
 context = WithinSessionEvaluation(paradigm=paradigm,
                                   datasets=datasets,
                                   random_state=42)
 
 pipelines = OrderedDict()
-pipelines['TS'] = make_pipeline(Covariances('oas'), TSclassifier())
-pipelines['CSP+LDA'] = make_pipeline(Covariances('oas'), CSP(8), LDA())
-pipelines['CSP+SVM'] = make_pipeline(Covariances('oas'), CSP(8), SVC())  #
-
-results = context.process(pipelines)
+pipelines['av+TS'] = make_pipeline(
+    mp.AverageCovariance(estimator='oas'), TSclassifier())
+pipelines['av+CSP+LDA'] = make_pipeline(
```

Comment:
is this what a CSP pipeline looks like with the multiband ?

Reply:
This is a CSP pipeline if you average the covariance matrices over all bands. Otherwise you pretty much have FBCSP already unless you decide to use all the resulting features

Reply:
This is very confusing for the user, and not equivalent numerically.